File: bot.js
const { Client, Intents } = require('discord.js');

const TOKEN = 'TOKEN';
const CHANNEL_ID = 'CHANNEL_ID';

const client = new Client({ intents: [513] });
 
client.on('ready', () => {
  console.log(`Botにログインしました！ ${client.user.tag}`);
});

client.on('messageCreate', async (message) => {
  if (!message.author.bot) {
    const channel = client.channels.cache.get(CHANNEL_ID);
    await channel.send(message.content);
    console.log(message.content);
  }
});

client.login(TOKEN);

File: bot.py
import discord

intents = discord.Intents.default()
intents.message_content = True

TOKEN = 'TOKEN'
CHANNELID = CHANNELID

client = discord.Client(intents=intents)

@client.event
async def on_ready():
    print(f'Botにログインしました！ {client.user}')

@client.event
async def on_message(message):
    if not message.author.bot:
        channel = client.get_channel(CHANNELID)
        await channel.send(message.content)
        print(message.content)

client.run(TOKEN)